contracts/src/v0.8/functions/dev/v1_X/FunctionsBilling.sol

```diff
@@ -174,6 +176,12 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
     uint256 estimatedGasReimbursement = juelsPerGas * executionGas;
     uint96 fees = uint96(donFee) + uint96(adminFee);
 
+    // Will return non-zero on chains that have this enabled
```

Comment:
Could we use branching logic to avoid running the logic above if on a network that needs specific gas handling?
Maybe a view function for `isChainSpecificGas`?